### ISSUE TITLE
version 5.12.1 wont run; incorrect install?

### ISSUE THREAD

**Issue by futagoza** [eslint/eslint#11296]
**Tell us about your environment**

* **ESLint Version:** 5.12.1
* **Node Version:** 10.15.0
* **npm Version:** 6.5.0
* **yarn Version:** 1.13.0

**What parser (default, Babel-ESLint, etc.) are you using?** babel-eslint

**Please show your full configuration:**

```js
"use strict";

module.exports = {

  extends: "futagozaryuu/node-v8",
  root: true,

};
```

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```bash
eslint .
```

**What did you expect to happen?**
Eslint to run

**What actually happened? Please include the actual, raw output from ESLint.**
Node threw an error about not being able to parse `eslint/lib/util/glob-utils.js`, and upon further inspection I found the file was a less then kb file with an unfinished header comment. Upon even further inspection, it seems the [published version](https://unpkg.com/eslint@5.12.1/lib/util/glob-utils.js) is fine, the problem only occurs after installing 5.12.1

**Are you willing to submit a pull request to fix this bug?**
Not sure what's wrong, so no. If I knew what was going on I'de be willing to fix it, but after a few attempts I've just rolled back to 5.12.0

**Comment by g-plane** [comment]
Have you re-installed and tried again?

**Comment by futagoza** [comment]
Yep, a dozen times over, both using yarn and npm

**Comment by g-plane** [comment]
Did you install globally or locally? Different installing methods could cause different behaviors.

**Comment by futagoza** [comment]
Always have installed locally, I never install globally 😆

**Comment by g-plane** [comment]
I just noticed that your command is `eslint .`, but I didn't know if was shell script or npm script.

**Comment by futagoza** [comment]
It's from an npm script: `npm run lint` or `yarn lint`

I only use the shell directly when I need to use Node, NPM or Yarn.

**Comment by futagoza** [comment]
The file `eslint/lib/util/glob-utils.js` seems to be truncated, so I think this issue isn't a matter of _how_ I use ESLint, but a matter of _install_.

To be honest, I'm thinking there's a high chance this is linked to some other problem; either with npm/yarn or my system, but because I'm unable to locate it, I thought the best place to report this issue would be here.

**Comment by futagoza** [comment]
I know it was, I mentioned it in the op 😉

**Comment by g-plane** [comment]
Can you clean the npm cache or Yarn cache then try again?

**Comment by futagoza** [comment]
Clearing both caches (NPM and Yarn) _and then_ doing a clean install using either seems to work fine.

Looks like ESLint 5.12.1 somehow got corrupted (more specifically the file `eslint/lib/util/glob-utils.js` seems to be truncated) when initially downloaded (via the install command) by both Yarn and NPM, although I'm not sure how that's possible, any ideas?
